### PR TITLE
Add gh actions workflows from PLAYGROUND

### PR DESCRIPTION
**Pull request by jgara**
No issues.

### CODE REVIEW

File: .github/workflows/tests-integration.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```
Comment (aforward) [inline .github/workflows/tests-integration.yml:14]: Can you confirm this must be Java 11 whereas the others are OK to be Java 14

**Review by aforward — COMMENTED** [review]
One comment.

File: .github/workflows/tests-integration.yml (hunk 1 above)
Comment (jgara) [reply .github/workflows/tests-integration.yml:14]: Yes. I can confirm that `mvn test` works fine with Java 14. But -- `mvn verify` needs Java 11.

File: .github/workflows/tests-integration.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: Run Unit Tests
```
Comment (aforward) [inline .github/workflows/tests-integration.yml]: `Run Integration Tests`

**Review by aforward — CHANGES_REQUESTED** [review]
Minor label change

**Review by aforward — APPROVED** [review]

**Pull request merged**